src/components/Lists/ListBase/ListBase.stories.tsx

```diff
@@ -17,35 +22,68 @@ import { Story } from "@storybook/react";
 
 export default {
     title: "Components/ListBase",
-    subcomponents: { ListItemBase, ListItemIconBase, ListItemTextBase, TypographyBase },
+    subcomponents: {
+        ListItemBase,
+        ListItemIconBase,
+        ListItemTextBase,
+        TypographyBase,
+        ButtonBase,
+        BoxBase,
+    },
     component: ListBase,
 };
 
 const simpleListItems = [
-    { icon: <StarIcon />, text: "Starred" },
-    { icon: <SendIcon />, text: "Sent" },
-    { icon: <DeleteIcon />, text: "Trash" },
-    { icon: <ReportIcon />, text: "Spam" },
-    { icon: <DraftsIcon />, text: "Drafts" },
+    { icon: <InboxIcon />, text: "Inbox", highlighted: true },
+    { icon: <StarIcon />, text: "Starred", highlighted: false },
+    { icon: <SendIcon />, text: "Sent", highlighted: false },
+    { icon: <DeleteIcon />, text: "Trash", highlighted: false },
+    { icon: <ReportIcon />, text: "Spam", highlighted: false },
+    { icon: <DraftsIcon />, text: "Drafts", highlighted: false },
 ];
 
 export const SimpleList: Story<ListBaseProps> = (args) => (
-    <ListBase {...args}>
-        <ListItemBase>
-            <ListItemIconBase color="secondary">
-                <InboxIcon />
-            </ListItemIconBase>
-            <ListItemTextBase primary="Inbox" color="secondary-main" />
-        </ListItemBase>
-
-        {simpleListItems.map((item, index) => (
-            <ListItemBase key={index}>
-                <ListItemIconBase>{item.icon}</ListItemIconBase>
-                <ListItemTextBase primary={item.text} />
-            </ListItemBase>
-        ))}
-    </ListBase>
+    <BoxBase width={360}>
+        <ListBase {...args}>
+            {simpleListItems.map((item, index) => (
+                <ListItemBase key={index} button highlighted={item.highlighted}>
+                    <ListItemIconBase>{item.icon}</ListItemIconBase>
+                    <ListItemTextBase primary={item.text} />
+                </ListItemBase>
+            ))}
+        </ListBase>
+    </BoxBase>
+);
+
+const replyLogoUrl = "https://res.cloudinary.com/dq7l8216n/image/upload/v1621347887/Reply_Logo.png";
+const fcodeAvatarUrl =
+    "https://res.cloudinary.com/dq7l8216n/image/upload/v1620235303/FCode-Avatar.png";
+
+const loggedInListItems = [
+    { icon: <InboxIcon />, text: "Inbox", highlighted: false },

```

Comment:
```suggestion
    { icon: <InboxIcon />, text: "Inbox", highlighted: true },

```
Should this be true?